### PR TITLE
pass all Flatpickr parameters to props.options

### PR DESCRIPTION
**Pull request by landvibe**
it resolves below warnings
```
bundle.js:10659 Warning: Unknown props `minDate`, `maxDate` on <input> tag. Remove these props from the element. For details, see https://fb.me/react-unknown-prop
```
```
warning.js:36Warning: Failed form propType: You provided a `value` prop to a form field without an `onChange` handler. This will render a read-only field. If the field should be mutable use `defaultValue`. Otherwise, set either `onChange` or `readOnly`. Check the render method of `DateTimePicker`
```

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -27,13 +27,13 @@ class DateTimePicker extends Component {
 
     const node = document.getElementById(id)
 
-    this.flatpickr = new Flatpickr(node, this.props)
+    this.flatpickr = new Flatpickr(node, this.props.options)
   }
 
   render() {
-    const { onChange, ...props } = this.props
+    // ignore onChange, options
```
Comment (coderhaoxin) [inline lib/index.js]: For most use cases, we should add `onChange` to `options`

File: example/index.js [hunk 2]
```diff
@@ -26,11 +26,13 @@ class App extends Component {
     return (
       <main>
         <Flatpickr data-enable-time className='test'
-          onChange={(_, str) => console.info(str)} />
+          options={{onChange: (_, str) => console.info(str)}} />
```
Comment (coderhaoxin) [inline example/index.js]: we should support pass prop `onChange` directly.

Comment (coderhaoxin) [inline example/index.js]: Sorry, I mean.

I want to use `Flatpickr` like this

```jsx
<Flatpickr onChange={...} options={...} />
```

For most use cases, maybe we don't need pass options.
We should `alias` `props.onChange` to `props.options.onChange`

**Comment by landvibe** [comment]
I set defaultValue manually because the textbox should show options.value initially

**Comment by landvibe** [comment]
I also moved value from options to props

File: lib/index.js [hunk 3]
```diff
@@ -26,20 +26,28 @@ class DateTimePicker extends Component {
...
     const { id } = this.state
-    // ignore onChange
     return (
-      <input {...props} id={id} />
+      <input {...props} id={id} defaultValue={defaultValue || value}/>
     )
   }
 }
 
+DateTimePicker.defaultProps = {
```
Comment (coderhaoxin) [inline lib/index.js]: I prefer

```js
static defaultProps = {
  options: {}
}
```

Comment (landvibe) [inline lib/index.js]: changed the position

**Comment by coderhaoxin** [comment]
Thanks !

published `v2.0.0`

**Pull request merged**